### ISSUE TITLE
Roadmap-Planning & Support for Angular5/6

### ISSUE THREAD

**Issue by MarcusMorba** [angular/flex-layout#708]
Do avoid a similar situation we had with Angular 4 and the FlexLayout Beta 10/11 I would like to know if for the future versions there will be support for Angular 5 and Angular 6 or if Angular 6 might get a requirement in the closer future.

Thanks in advance.

**Comment by CaerusKaru** [comment]
Our current plan is to support Angular 5 on the `latest` tag on NPM (ie maintain 5.0.0-beta.14 and if there are emergency patches _maybe_ we would release another 5.x beta). However, our immediate plan is to cut a 6.x release on the `next` tag on NPM, likely some time today.

It's currently not possible to support Angular 5 and 6 simultaneously in the same build due to various factors (including switching our Injectable API, upgrading to RxJS 6, and other build changes).

tl;dr We will publish a 6.x beta release very soon, but it will be on the next tag, so it's totally opt-in. We will still have the 5.x release on latest until Angular v6 goes to final
